sunpy/net/dataretriever/sources/goes.py

```diff
@@ -81,7 +82,9 @@ class XRSClient(GenericClient):
 
     @property
     def info_url(self):
-        return 'https://umbra.nascom.nasa.gov/goes/fits'
+        return ("\n <13: https://umbra.nascom.nasa.gov/goes/fits \n"
```

Comment:
Why does this now need to start with a new line?

Reply:
It doesn't I wanted to see if I could make it all start on a new line to prevent the indent but it had no affect.

Reply:
REmoved.